test/unit/api/metadata/handler.test.js

```diff
@@ -100,10 +100,92 @@ Test('metadata handler', (handlerTest) => {
+        services: Joi.array().required()
+      }
+      const expectedStatus = 200
+      const expectedServices = []
+
+      // Act
+      const {
+        responseBody,
+        responseCode
+      } = await unwrapResponse((reply) => Handler.getHealth(createRequest({}, { query: { simple: '' } }), reply))
+
+      // Assert
+      const validationResult = Joi.validate(responseBody, expectedSchema)
+      test.equal(validationResult.error, null, 'The response matches the validation schema')
+      test.deepEqual(responseCode, expectedStatus, 'The response code matches')
+      test.deepEqual(responseBody.services, expectedServices, 'The sub-services are empty')
+      test.end()
+    })
+
+    healthTest.test('get simple health status with query string "simple=true"', async test => {
+      // Arrange
+      const expectedSchema = {
+        status: Joi.string().valid('OK').required(),
+        uptime: Joi.number().required(),
+        startTime: Joi.date().iso().required(),
+        versionNumber: Joi.string().required(),
+        services: Joi.array().required()
+      }
+      const expectedStatus = 200
+      const expectedServices = []
+
+      // Act
+      const {
+        responseBody,
+        responseCode
+      } = await unwrapResponse((reply) => Handler.getHealth(createRequest({}, { query: { simple: true } }), reply))
+
+      // Assert
+      const validationResult = Joi.validate(responseBody, expectedSchema)
+      test.equal(validationResult.error, null, 'The response matches the validation schema')
+      test.deepEqual(responseCode, expectedStatus, 'The response code matches')
+      test.deepEqual(responseBody.services, expectedServices, 'The sub-services are empty')
+      test.end()
+    })
+
+    healthTest.test('get detailed health status with query string "simple=false"', async test => {
+      // Arrange
+      axios.get.withArgs(Config.ENDPOINT_HEALTH_URL).resolves({ data: { status: 'OK' } })
+      const expectedSchema = {
+        status: Joi.string().valid('OK').required(),
+        uptime: Joi.number().required(),
+        startTime: Joi.date().iso().required(),
+        versionNumber: Joi.string().required(),
+        services: Joi.array().required()
+      }
+      const expectedStatus = 200
+      const expectedServices = [
+        { name: 'broker', status: 'OK' },
+        { name: 'participantEndpointService', status: 'OK' }
+      ]
+
+      // Act
+      const {
+        responseBody,
+        responseCode
+      } = await unwrapResponse((reply) => Handler.getHealth(createRequest({}, { query: { simple: false } }), reply))
+
+      // Assert
+      const validationResult = Joi.validate(responseBody, expectedSchema)
+      test.equal(validationResult.error, null, 'The response matches the validation schema')
+      test.deepEqual(responseCode, expectedStatus, 'The response code matches')
+      test.deepEqual(responseBody.services, expectedServices, 'The sub-services are correct')
+      test.end()
+    })
+*/
```

Comment:
That commented-out code makes sense here ?

Reply:
removed